addons/binding/org.openhab.binding.logreader/README.md

```diff
@@ -0,0 +1,110 @@
+# Log Reader Binding
+
+This binding reads and analyzes log files. Search patterns are fully configurable, therefore different kind of log files should be possible to monitor by this binding.
+When certain log events is recognized, openHAB rules can be used to send notification about the event e.g by email for further analysis.
+
+## Supported Things
+
+This binding supports one ThingType: `reader`.
+A reader supports 3 separate channels; One for errors, one for warnings and one custom channel for other purposes.
+If you need more channels, you can always add another Thing.
```

Comment:
I'd prefer to remove this sentence - if it is important, it rather shows that there is something wrong with the modelling, so it should be in your own interest to remove it ;-)

Reply:
I didn't remove it yet as I like to study little bit what it means to support dynamic channels. It seems to cause some refactoring as now regex patterns etc are in binding configuration (not per channel). 

I guess that 2.3 door is closing soon?

Reply:
You can remove this sentence without refactoring anything right now.
I also thought on how to best generify the channels, but as you say it isn't easy as they always share the configuration, which is within the binding. So I am fine with leaving it as it is, but I'd still prefer this sentence from the doc removed.

Reply:
Removed